docs/third-party-developers/extensibility/checkout-block/available-filters.md

```diff
@@ -212,29 +186,10 @@ __experimentalRegisterCheckoutFilters( 'automatic-coupon-extension', {
 | -------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------- |
 | ![image](https://user-images.githubusercontent.com/5656702/123768988-bc55eb80-d8c0-11eb-9262-5d629837706d.png) | ![image](https://user-images.githubusercontent.com/5656702/124126048-2c57a380-da72-11eb-9b45-b2cae0cffc37.png) |
 
-### Hide a snackbar notice containing a certain string
```

Comment:
Could we document the `showApplyCouponNotice` and `showRemoveCouponNotice` filters in this PR?

Reply:
Added!